core/storage/suggestion/gae_models.py

```diff
@@ -0,0 +1,81 @@
+# coding: utf-8
```

Comment:
One other thought: feedback_models isn't that big, so it may be simpler to just append this class there instead. What do you think @BenHenning?